.github/workflows/Tests.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Regression testing
+
+# https://docs.github.com/en/actions/using-jobs/using-concurrency
```

Comment:
Rather than linking to the concurrency documentation, can you explain why we want / need the concurrency statement?

Reply:
Done in https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pr/pull/18/commits/14838d309728f06c81f4af8718eb55202d320987.